var/spack/repos/builtin/packages/vtk/package.py

```diff
@@ -293,6 +297,8 @@ def cmake_args(self):
                 )
             if spec.satisfies("@9.2:"):
                 cmake_args.append("-DVTK_MODULE_USE_EXTERNAL_VTK_verdict:BOOL=OFF")
+            if spec.satisfies("@9.3:"):
+                cmake_args.append("-DVTK_MODULE_USE_EXTERNAL_VTK_fast_float:BOOL=OFF")
```

Comment:
We also added this in for paraview 5.12.

```python
cmake_args.append("-DVTK_MODULE_USE_EXTERNAL_VTK_token:BOOL=OFF")
```

Reply:
@kwryankrattiger, #44966 added vtk 9.3.1 meanwhile. Do you say that
```py
cmake_args.append("-DVTK_MODULE_USE_EXTERNAL_VTK_token:BOOL=OFF")
```
should be added as well for `vtk@9.3:`?

Could you explain this to an uninitiated that it's clear what to do?

Turning off EXTERNAL_VTK_token may not be needed not needed (as vtk@9.3.1 compiles),
Is it better to add it nonetheless?

I plan to resolve the conflicts and rebase to the develop if possible.

The only parts that would remain are the python version dependencies for 9.2 and 9.3, AFAICS.

Can you approve these python version limits for 9.2 and 9.3 then?